Bindings/Java/Matlab/Utilities/createActuatorsFile.m

```diff
@@ -1,42 +1,14 @@
-%createActuatorsFile   make and Print an OpenSim Actuator File from a Model
-%
-%  createActuatorsFile makes a template actuators file that can
-%  be used in Static Optimization, RRA and CMC. This identifies the
-%  coordinates that are connected to ground and places point or torque
-%  actuators on translational or rotational coordinates, respectively. All
-%  other coordiantes will get coordinate actuators. Any constrained 
-%  coordinates will be ignored.
-
-% ----------------------------------------------------------------------- %
-% The OpenSim API is a toolkit for musculoskeletal modeling and           %
-% simulation. See http://opensim.stanford.edu and the NOTICE file         %
-% for more information. OpenSim is developed at Stanford University       %
-% and supported by the US National Institutes of Health (U54 GM072970,    %
-% R24 HD065690) and by DARPA through the Warrior Web program.             %
-%                                                                         %
-% Copyright (c) 2005-2016 Stanford University and the Authors             %
-% Author(s): James Dunne                                                  %
-%                                                                         %
-% Licensed under the Apache License, Version 2.0 (the "License");         %
-% you may not use this file except in compliance with the License.        %
-% You may obtain a copy of the License at                                 %
-% http://www.apache.org/licenses/LICENSE-2.0.                             %
-%                                                                         %
-% Unless required by applicable law or agreed to in writing, software     %
-% distributed under the License is distributed on an "AS IS" BASIS,       %
-% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
-% implied. See the License for the specific language governing            %
-% permissions and limitations under the License.                          %
-% ----------------------------------------------------------------------- %
-
-% Author: James Dunne, Tom Uchida, Chris Dembia, 
-% Ajay Seth, Ayman Habib, Shrinidhi K. Lakshmikanth, Jen Hicks.
-
 function createActuatorsFile(varargin)
+%  Make a template actuators file
+%  Function identifies the coordinates that are connected to ground and 
+%  places point or torque actuators on translational or rotational 
+%  coordinates, respectively. All other coordinates will get coordinate 
+%  actuators. Any constrained coordinates will be ignored.
```

Comment:
As a matter of practice, it would be good if each function provided its usage at the end of the function comments.
e.g. `Usage: createActuatorsFile((optional) modelFilePath)`